www/docs/en/dev/cordova/events/events.md

```diff
@@ -323,6 +323,16 @@ function onBackKeyDown() {
 }
 ```
 
+### Windows Quirks
+
+Throw an error in a `backbutton` callback to force the default behavior:
```

Comment:
For a single page app, it will result in exiting? Shall we add that to the doc comment.